UK/Area/Scottish Upper/Area.prf

```diff
@@ -1,14 +1,14 @@
 Settings	SettingsfilePLUGINS	\..\..\Data\Settings\Local\Scottish\PX_AC_Plugins.txt
 Settings	SettingsfilePROFILE	\..\..\Data\Settings\Local\Scottish\PX_AC_Profiles_No-CPDLC.txt
-Settings	SettingsfileSCREEN	\..\..\Data\Settings\AC\AC_Screen.txt
+Settings	SettingsfileSCREEN	\..\..\Data\Settings\Screen.txt
 Settings	SettingsfileVOICE	\..\..\Data\Settings\Voice_Scottish.txt
 Settings	Settingsfile	\..\..\Data\Settings\AC\AC_General.txt
-Settings	SettingsfileARR	\..\..\Data\Settings\Lists.txt
+Settings	SettingsfileARR	\..\..\Data\Settings\Lists_SMR.txt
 Settings	SettingsfileCONFLICT	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileDEP	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileFP	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileSEL	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileSIL	\..\..\Data\Settings\Lists.txt
+Settings	SettingsfileDEP	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileFP	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileSEL	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileSIL	\..\..\Data\Settings\Lists_SMR.txt
```

Comment:
To fix

Reply:
resolved via https://github.com/VATSIM-UK/uk-controller-pack/pull/518/commits/6384cbe3e0bc8050daa8c726ba974461cc579d70